src/config.ts

```diff
@@ -41,10 +44,34 @@ if (
     );
 }
 
+if (!INVENTORY_TEZOS_SECRET_KEY) {
```

Comment:
the secret key and public key hash will be the same whether we deploy an inventory or warehouse contract, it's just a way for the user (owner of both contract) to authenticate with the tezos node. So I think we can remove those 2 environment variables